tests/k8s/test_cronjob.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+# Copyright 2017-2019 The FIAAS Authors
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import mock
+import pytest
+
+from k8s.client import NotFound
+from k8s.models.common import ObjectMeta
+from k8s.models.cronjob import CronJob
+from k8s.models.job import Job, JobSpec
+from k8s.models.pod import Container, LocalObjectReference, PodTemplateSpec, PodSpec
+
+
+NAME = "my-name"
+NAMESPACE = "my-namespace"
+
+
+@pytest.mark.usefixtures("k8s_config")
+class TestCronJobs(object):
+    def test_create_blank_job(self):
+        object_meta = ObjectMeta(name=NAME, namespace=NAMESPACE)
+        cronjob = CronJob(metadata=object_meta)
+        assert cronjob.as_dict()[u"metadata"][u"name"] == NAME
+
+    def test_created_if_not_exists(self, post, api_get):
+        api_get.side_effect = NotFound()
+        cronjob = _create_default_job()
+        call_params = cronjob.as_dict()
+        post.return_value.json.return_value = call_params
+
+        assert cronjob._new
+        cronjob.save()
+        assert not cronjob._new
+
+        pytest.helpers.assert_any_call(post, _uri(NAMESPACE), call_params)
+
+        
+def _create_default_cronjob():
+    labels = {"test": "true"}
```

Comment:
Codacy found an issue: [local variable 'labels' is assigned to but never used (F841)](https://app.codacy.com/manual/fiaas/k8s/pullRequest?prid=5919885)